application/views/interface_assets/footer.php

```diff
@@ -2383,9 +2383,14 @@ className: 'buttons-clear',
 
 <script>
 function viewQsl(picture, callsign) {
+            <?php 
+            $CI =& get_instance();
+            $CI->load->model('Qsl_model');
```

Comment:
There are several places in the `views` you've changed where you load a `model` in a `view`. We would like to stay in the classic MCV pattern which means you should load the models in the controller and give `$data` to the view 😊

Reply:
i will change this on next commit, but not on controller.